yt/python/contrib/python-requests/requests/__init__.py

```diff
@@ -87,8 +87,8 @@ def check_compatibility(urllib3_version, chardet_version, charset_normalizer_ver
     if chardet_version:
         major, minor, patch = chardet_version.split('.')[:3]
         major, minor, patch = int(major), int(minor), int(patch)
-        # chardet_version >= 3.0.2, < 5.0.0
-        assert (3, 0, 2) <= (major, minor, patch) < (5, 0, 0)
+        # chardet_version >= 3.0.2, <= 5.2.0
+        assert (3, 0, 2) <= (major, minor, patch) <= (5, 2, 0)
```

Comment:
Let use the condition from the latest requests version: https://github.com/psf/requests/blob/main/src/requests/__init__.py#L79

Reply:
A bit risky, but ok:)

I've updated the pr.